lib/plugins/aws/package/compile/events/alb/lib/listenerRules.js

```diff
@@ -75,12 +75,13 @@ module.exports = {
       }
       Actions.push(forwardAction);
 
-      const Conditions = [
-        {
+      const Conditions = [];
+      if (event.conditions.path) {
```

Comment:
Should we require either `path` or `host`?

If so, I'd add validation clause, so we crash with meaningful error if none of are provided

Reply:
Good point. Let me add that in as well

Reply:
I looked at the documentation and also tried it directly in the alb console. It looks like the way it works is that you need at least 1 condition set and it has no limits/restrictions on what condition. therefore i have tweaked the code to match that behavior and validation